src/components/calculator.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+class Calculate extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: 0,
+    };
+  }
+
+  render() {
+    const { total } = this.state;
+
+    return (
+      <div className="calculator-box">
+        <table id="calculator" cellSpacing="0">
+          <tbody id="calculator-body">
+            <tr className="result-box">
+              <td colSpan="4" className="result">
+                <span>{total}</span>
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <button type="button" className="no-color" onClick={this.handPress}> AC </button>
+              </td>
+              <td>
+                <button type="button" className="no-color" onClick={this.handPress}> +/- </button>
+              </td>
+              <td>
+                <button type="button" className="no-color" onClick={this.handPress}> % </button>
+              </td>
+              <td>
+                <button type="button" className="color" onClick={this.handPress}> + </button>
```

Comment:
- [ ] From the design provided this button should be the divide sign, not plus. Kindly fix that.

  If you do not have the divide sign on your PC you can use the HTML entity code for this: **`&#xf7;`**

Reply:
thank you